ansible/archival-snapshots/resources/init.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+## Enable strict error handling, command tracing, and pipefail
+set -euxo pipefail
+
+FOREST_CLI="/home/archie/forest-v0.16.3/forest-cli"
+FOREST_TOOL="/home/archie/forest-v0.16.3/forest-tool"
+
+# Setting DEBIAN_FRONTEND to ensure non-interactive operations for APT
+export DEBIAN_FRONTEND=noninteractive
+
+CURRENT_SNAPSHOT=$(aws --profile prod --endpoint "$ENDPOINT" s3 ls "s3://forest-archive/mainnet/lite/" | sort | tail -n 1 | awk '{print $NF}')
+CURRENT_EPOCH=$(echo "$CURRENT_SNAPSHOT" | awk -F'_' '{gsub(/[^0-9]/, "", $6); print $6}')
+
+echo "$CURRENT_EPOCH"
+
+LATEST_EPOCH=$($FOREST_CLI sync status | grep "Height:" | awk '{print $2}')
```

Comment:
You can't rely on a Forest node already running on the server.

Reply:
handled will be starting forest if its not running already